### ISSUE TITLE
What if the ground truth contains NaNs?

### ISSUE THREAD

**Issue by ihopethiswillfi** [felixykliu/NAOMI#1]
Hi
From the paper it seems that in all of the 3 examples, the original input data is always data for which all of the values are already known.  I.e. the ground truth never contains NaNs.

Looking at the code confirms this.  The code does not account for potential missing data in the ground truth.

Would you say this is a limitation of the NAOMI algorithm itself, or of its implementation?

**Comment by ATTERasd** [comment]
HI.
I agree with your point.
The code is not applicable when the ground truth of missing values are not given.
NAOMI requires the ground truth as the model input at the pretraining process and the conponents of the loss function.
